install/installer/pkg/components/proxy/deployment.go

```diff
@@ -119,6 +119,30 @@ func deployment(ctx *common.RenderContext) ([]runtime.Object, error) {
 		return nil
 	})
 
+	env := common.CustomizeEnvvar(ctx, Component, common.MergeEnv(
+		common.DefaultEnv(&ctx.Config),
+		[]corev1.EnvVar{{
+			Name:  "PROXY_DOMAIN",
+			Value: ctx.Config.Domain,
+		}},
+	))
+
+	_ = ctx.WithExperimental(func(cfg *experimental.Config) error {
+		if cfg.WebApp != nil {
+			env = append(env, corev1.EnvVar{
+				Name:  "CONFIGCAT_SDK_KEY",
+				Value: cfg.WebApp.ConfigcatKey,
+			}, corev1.EnvVar{
+				Name:  "CONFIGCAT_BASE_URL",
+				Value: cfg.WebApp.ConfigcatBaseUrl,
+			}, corev1.EnvVar{
+				Name:  "CONFIGCAT_POLL_INTERVAL",
+				Value: cfg.WebApp.ConfigcatPollInterval,
+			})
+		}
+		return nil
+	})
```

Comment:
There's a function in `common.ConfigcatEnv()` which already sets the SDK key. Perhaps we can extend it with the extra properties and then re-use in other components which instantiate ConfigCat

Reply:
Updated to add it to `common`, see diff [here](https://github.com/gitpod-io/gitpod/compare/1eeb694d1875331e35342d688346075f9c9cfa78..f3dae22458e64ccbdfda6e34283788b30cf0d300)

Reply:
LGTM!